types/validator/index.d.ts

```diff
@@ -271,10 +272,17 @@ declare namespace validator {
      */
     function isByteLength(str: string, options?: IsByteLengthOptions): boolean;
 
+    interface IsCreditCardOptions {
+        /**
+         * @default undefined
+         */
+        provider?: 'amex' | 'dinersclub' | 'discover' | 'jcb' | 'mastercard' | 'unionpay' | 'visa';
```

Comment:
docs mentions:

> or blank 

can you check this with source what does it mean? e.g. '' is a blank, thanks!

Reply:
It means any falsy value. I've added an empty string as a type for the 'blank' value.